pixeltable/catalog/catalog.py

```diff
@@ -1018,6 +1060,8 @@ def create_table(
 
         @retry_loop(for_write=True)
         def create_fn() -> tuple[UUID, bool]:
+            import pixeltable.metadata.schema
```

Comment:
Why the unusual import pattern? Everywhere else in the code this is imported as `schema`

Reply:
Done.